workflow-ui/core-android/src/androidTest/java/com/squareup/workflow1/ui/WorkflowViewStubLifecycleTest.kt

```diff
@@ -29,6 +53,7 @@ internal class WorkflowViewStubLifecycleTest {
   @Test fun stop_then_resume() {
     assertThat(scenario.state).isEqualTo(RESUMED)
     scenario.onActivity {
+      it.update(LeafRendering("initial"))
```

Comment:
This call was removed from the test activity's `onCreate` method to support other tests, so now each test that needs that previous behavior has to do the initial update itself. This makes the tests more clear anyway.